admin-dev/themes/new-theme/js/pages/order/view/order-discounts-refresher.js

```diff
@@ -34,7 +34,7 @@ export default class OrderDiscountsRefresher {
   }
 
   refresh(orderId) {
-    $.getJSON(this.router.generate('admin_orders_get_discounts', {orderId}))
+    $.ajax(this.router.generate('admin_orders_get_discounts', {orderId}))
```

Comment:
What is the reason behind this. The result is no more a json? :thinking:

Reply:
@PierreRambaud it never has actually! If you check [2 lines further](https://github.com/PrestaShop/PrestaShop/blob/1.7.7.x/admin-dev/themes/new-theme/js/pages/order/view/order-discounts-refresher.js#L39), you can see that we are using `replaceWith(response)` directly.

Reply:
trololololo